azext_edge/edge/commands_namespaces.py

```diff
@@ -0,0 +1,1971 @@
+        asset_name=asset_name,
+        instance_name=instance_name,
+        instance_resource_group=instance_resource_group,
+        event_name=event_name,
+        asset_type="custom",
+        datapoint_name=datapoint_name,
+        data_source=data_source,
+        custom_configuration=custom_configuration,
+        replace=replace,
+        **kwargs
+    )
+
+
+# TODO: not exposed for now but this will be supported in the near future
+def add_namespace_opcua_asset_event_point(
+    cmd,
+    asset_name: str,
+    instance_name: str,
+    instance_resource_group: str,
+    event_name: str,
+    datapoint_name: str,
+    data_source: str,
+    queue_size: Optional[int] = None,
+    sampling_interval: Optional[int] = None,
+    replace: Optional[bool] = False,
+    **kwargs
+) -> dict:
+    return NamespaceAssets(cmd).add_event_datapoint(
+        asset_name=asset_name,
+        instance_name=instance_name,
+        instance_resource_group=instance_resource_group,
+        event_name=event_name,
+        asset_type=DeviceEndpointType.OPCUA.value,
+        datapoint_name=datapoint_name,
+        data_source=data_source,
+        queue_size=queue_size,
+        sampling_interval=sampling_interval,
+        replace=replace,
+        **kwargs
+    )
+
+
+def list_namespace_asset_event_points(
+    cmd,
+    asset_name: str,
+    instance_name: str,
+    instance_resource_group: str,
+    event_name: str
+) -> List[dict]:
+    return NamespaceAssets(cmd).list_event_datapoints(
+        asset_name=asset_name,
+        instance_name=instance_name,
+        instance_resource_group=instance_resource_group,
+        event_name=event_name
+    )
+
+
+def remove_namespace_asset_event_point(
+    cmd,
+    asset_name: str,
+    instance_name: str,
+    instance_resource_group: str,
+    event_name: str,
+    datapoint_name: str,
+    **kwargs
+) -> dict:
+    return NamespaceAssets(cmd).remove_event_datapoint(
+        asset_name=asset_name,
+        instance_name=instance_name,
+        instance_resource_group=instance_resource_group,
+        event_name=event_name,
+        datapoint_name=datapoint_name,
+        **kwargs
+    )
+
+
+# STREAM COMMANDS
+def add_namespace_custom_asset_stream(
+    cmd,
+    asset_name: str,
+    instance_name: str,
+    instance_resource_group: str,
+    stream_name: str,
+    stream_custom_configuration: Optional[str] = None,
+    stream_destinations: Optional[str] = None,
+    replace: Optional[bool] = False,
+    **kwargs
+) -> dict:
+    return NamespaceAssets(cmd).add_stream(
+        asset_name=asset_name,
+        instance_name=instance_name,
+        instance_resource_group=instance_resource_group,
+        stream_name=stream_name,
+        asset_type="custom",
+        stream_custom_configuration=stream_custom_configuration,
+        stream_destinations=stream_destinations,
+        replace=replace,
+        **kwargs
+    )
+
+
+def add_namespace_media_asset_stream(
```

Comment:
from the meeting - seems to be consensus that I can split this + update up by task type